doc/contributing/maintaining-types-for-nodejs.md

```diff
@@ -61,3 +61,12 @@ afterwards.
 There is an ongoing effort to add additional markdown constraints and
 then update the flow in order to be able to generate a better
 JSON output.
+
+[doc/api]: https://github.com/nodejs/node/tree/master/doc/api
+[documentation-style-guide]: https://github.com/nodejs/node/blob/master/doc/README.md#documentation-style-guide
+[tools/doc]: https://github.com/nodejs/node/tree/master/tools/doc
+[nodejs.org]: https://nodejs.org/en/docs/
+[next-10]: https://github.com/nodejs/next-10/blob/main/meetings/summit-nov-2021.md#suitable-types-for-end-users
+[suitable types for end users]: https://github.com/nodejs/node/blob/master/doc/contributing/technical-priorities.md#suitable-types-for-end-users
+[Flow]: https://flow.org/
+[TypeScript]: https://www.typescriptlang.org/
```

Comment:
Re-order in ASCII order, and use `HEAD` instead of `master` (in case the default branch name ever changes).
```suggestion
[Flow]: https://flow.org/
[TypeScript]: https://www.typescriptlang.org/
[doc/api]: https://github.com/nodejs/node/tree/HEAD/doc/api
[documentation-style-guide]: https://github.com/nodejs/node/blob/HEAD/doc/README.md#documentation-style-guide
[next-10]: https://github.com/nodejs/next-10/blob/HEAD/meetings/summit-nov-2021.md#suitable-types-for-end-users
[nodejs.org]: https://nodejs.org/en/docs/
[suitable types for end-users]: https://github.com/nodejs/node/blob/HEAD/doc/contributing/technical-priorities.md#suitable-types-for-end-users
[tools/doc]: https://github.com/nodejs/node/tree/HEAD/tools/doc
```